packages/data/abstract/prophecies_repository/lib/src/prophecy.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:meta/meta.dart';
+import 'package:change_model/change_model.dart';
+import 'package:json_annotation/json_annotation.dart';
+import 'package:prophecy_model/prophecy_model.dart';
+
+part 'prophecy.g.dart';
+
+@JsonSerializable()
+class ProphecyEntity {
+  ProphecyModel model;
+  final List<ChangeModel> changes;
```

Comment:
I'm actually not sure we'd need *changes* to live inside *prophecy*. Any change might be applied to any prophecy basically (according to some rules that will be later defined). We can persist those changes and remember what prophecy use it, but it would be more efficient to store changes separately and then just remember its ids in the prophecy entity.

Its fine for now, but we will certainly would need to refactor this soon.

Reply:
Changes field will be changed from algorithm/app.